Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/ExposureWindowRiskLevelConfig.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.appconfig
+
+import de.rki.coronawarnapp.server.protocols.internal.v2.AppConfigAndroid
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
+
+interface ExposureWindowRiskLevelConfig {
+    val minutesAtAttenuationFilters: List<RiskCalculationParametersOuterClass.MinutesAtAttenuationFilter>
+    val minutesAtAttenuationWeights: List<RiskCalculationParametersOuterClass.MinutesAtAttenuationWeight>
+    val transmissionRiskLevelEncoding: RiskCalculationParametersOuterClass.TransmissionRiskLevelEncoding
+    val transmissionRiskLevelFilters: List<RiskCalculationParametersOuterClass.TrlFilter>
+    val transmissionRiskLevelMultiplier: Double
+    val normalizedTimePerExposureWindowToRiskLevelMapping:
```

Comment:
is this normalizedTimePerEWToRiskLevelMapping or normalizedTimePerDayToRiskLevelMapping?
see https://github.com/corona-warn-app/cwa-app-android/blob/519b80294bdef929aeced74d633219a145bb31d3/Server-Protocol-Buffer/src/main/proto/internal/v2/risk_calculation_parameters.proto#L14-L16

Reply:
its `normalizedTimePerEWToRiskLevelMapping`, ive extended `EW` to `ExposureWindow` as i was not quite sure why it was abbreviated

Reply:
the mapping of the properties is done here: https://github.com/corona-warn-app/cwa-app-android/pull/1546/files#diff-83ea3316e5ed20702e7a6fca8f8dcf8cfedb88352276ee34bb30e2c1d8323cc6R26